frontend/src/js/controllers/hostedChallengeCtrl.js

```diff
@@ -32,6 +34,22 @@
                             for (var j=0; j<data.results.length; j++){
                                 vm.challengeList.push(data.results[j]);
                             }
+                            
+                            vm.currentList = data.results;
+                            for (var i in vm.currentList) {
+
+                                var descLength = vm.currentList[i].description.length;
+                                if (descLength >= 50) {
+                                    vm.currentList[i].isLarge = "...";
+                                } else {
+                                    vm.currentList[i].isLarge = "";
+                                }
```

Comment:
@yashdusing I don't think we need the code from line 41 - 46. Am I missing something here?